<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardCell</class>
 <widget class="QWidget" name="StandardCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>266</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>266</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#container {
    border: 1px solid rgba(52, 52, 52, 0.15);
    border-radius: 10px;
}

#documentBack {
    background-color: #EEF4F6;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
}

#documentType {
    background-color: transparent;
}

#menuButton {
    background-color: transparent;
}

#titleLabel {
    font-family: Open Sans;
    font-weight: 600;
    line-height: 24px;
    letter-spacing: 0%;
    font-size: 16px;
    color: black;
}

#imageContainer {
	background: transparent;
	border: none;
}

#menuButton {
	background: transparent;
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QWidget" name="documentBack" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>306</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>306</width>
          <height>200</height>
         </size>
        </property>
        <widget class="QPushButton" name="menuButton">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>10</y>
           <width>24</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../Resources/Resources.qrc">
           <normaloff>:/images/Automations/cell_header_menu.png</normaloff>:/images/Automations/cell_header_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="imageContainer">
         <property name="geometry">
          <rect>
           <x>24</x>
           <y>30</y>
           <width>260</width>
           <height>171</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>171</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>171</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../Resources/Resources.qrc">
           <normaloff>:/images/Reviews/document.png</normaloff>:/images/Reviews/document.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>260</width>
           <height>171</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Standard Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
